tests/js/spec/views/ruleBuilder.spec.jsx

```diff
@@ -77,13 +77,13 @@ describe('RuleBuilder', function () {
     await tick();
     wrapper.update();
 
-    const add = wrapper.find('Button');
+    const add = wrapper.find('AddButton');
```

Comment:
Select fields that allow for multiple selection will stay open upon selection (old behavior was to close the menu). Any button inside the menu will still be selectable. In this test, we're trying to select the `AddButton` at the end of the form, so we can be more specific and refer to it directly.